tests/next_tests/iterator_tests/test_with_toy_connectivity.py

```diff
@@ -14,6 +14,19 @@
 
 import numpy as np
 import pytest
+from next_tests.toy_connectivity import (
```

Comment:
This is an unrelated change, that I thought I need. Should be extracted into a separate PR.

Reply:
Maybe if the reviewer agrees, we can keep it? There is the `functional` -> `next` commit inbetween and a simple `revert` + `cherry-pick` doesn't work...